task/tkn-bundle/0.1/tkn-bundle.yaml

```diff
@@ -115,6 +115,8 @@ spec:
       value: $(params.CONTEXT)
     - name: IMAGE
       value: $(params.IMAGE)
+    - name: STEPS_IMAGE
```

Comment:
I think this should be defined in the step before this?

Reply:
Thanks for catching this! Fixed.